### ISSUE TITLE
Missing support for package name

### ISSUE THREAD

**Issue by jasongaare** [infinitered/flame#9]
## Issue

When the package name is not equivalent to `com.[appDisplayName]`, paths to android native files are incorrect

## How upgrade helper handles it

React Native Upgrade Helper supports two inputs:

1. App Name
2. Package Name 

<img width="558" alt="Screenshot 2023-10-05 at 3 32 41 PM" src="https://github.com/infinitered/flame/assets/15896334/bf95f5d6-179d-4bac-8981-5d17213962a1">

## Actual Result with Flame

Best I can tell, [the code found here here](https://github.com/infinitered/flame/blob/df7dfc8251ffea71f5735667560d1f64288225f2/src/react-native/fetchRNAppInfo.ts#L40-L51) (which is replacing RNDiff with the app name) is missing package name support for situation like this

This results in "File Not Found" errors, because the file path is incorrectly constructed

```sh
↠ Skipping: ./android/app/src/main/java/com/companycam/MainActivity.java (file not found)
```

## Expected Result with Flame

with package name "com.agilx.companycam" we'd replace "com.rndiff" 

```sh
■ File:   ./android/app/src/main/java/com/agilx/companycam/MainActivity.java

Upgrade Helper diff: ...
```

I'd love to take a look at this soon-ish to see if there's a quick solution. Maybe a field in `app.json` for a package name?

**Comment by jamonholmgren** [comment]
@jasongaare Yeah, good point. I'd like to make Flame much more resilient to files not being in predictable places, in general.

Would love some help with this!

**Comment by huyvovan0510** [comment]
I am also facing this issue, and the workaround is to change the displayName in the app.json file to allow Flame AI to follow the path. In your case, just change displayName to "displayName": "agilx/companycam". I hope this will help you solve this issue, @jasongaare.
